### DISCUSSION TITLE
Standup (Wednesday, November 2nd)

### DISCUSSION THREAD

**Discussion by antonieto** [BBGTC/food-bank-app#45] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Implemented #35 

**What are your priorities for the next 24 hours?**:
- Open a PR for #35
 - Define frontend folder structure

**Blockers**:
 - Nothing

**Shoutouts**:
 - @BrandonMagana for cracking the coding interview

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on changing themes and styles to new version

**What are your priorities for the next 24 hours?**:
 - Complete components with new system

**Blockers**:
 - new rneui works weird for custom components

**Shoutouts**:
 - @CarlosCRG19 because I love him

**Comment by antonieto** [comment]
# @CarlosCRG19 

What I did in the last 24 hours?
- Finished personal Django project

What I'll do in the next 24 hours?
- work on tests

Blockers:
- was not able to finish tests on time due to lack of time and delay on grasping some Django concepts

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Understood basics of django and DRF
 - Started working on #37 

**What are your priorities for the next 24 hours?**:
 -  Discuss about backend folder structure with @antonieto 
 -  Keep working on models and serializers

**Blockers**:
 - None

**Shoutouts**:
 - @Haziel for his 21st birthday
